packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -35,7 +35,7 @@ import CommentForm from './comment-form';
  * @param {Function} props.onAddReply       - The function to add a reply to a comment.
  * @param {Function} props.onCommentDelete  - The function to delete a comment.
  * @param {Function} props.onCommentResolve - The function to mark a comment as resolved.
- * @return {React.ReactNode} The rendered Comments component.
+ * @return {JSX.Element} The rendered Comments component.
```

Comment:
Why did this need to change? Just curious? What are we using elsewhere?

Reply:
@ellatrix - Actually it was updated automatically when running the prettier command. Now replaced it back.